resources/js/Pages/Galleries/hooks/useGalleryDrafts.ts

```diff
@@ -131,6 +138,17 @@ export const useGalleryDrafts = (givenDraftId?: number, disabled?: boolean): Gal
         setReachedLimit(false);
     };
 
+    const deleteExpiredDrafts = async (): Promise<void> => {
+        const thirtyDaysAgo = new Date().getTime() - DRAFT_TTL_DAYS * 86400 * 1000;
```

Comment:
granted this variable name may not be the best since `DRAFT_TTL_DAYS` is configurable 🤔 maybe rename that to `thresholdDaysAgo` or something generic

Reply:
Nice, nice. Will fix that :)